### DISCUSSION TITLE
Global events/hooks

### DISCUSSION THREAD

**Discussion by BKSalman** [hecrj/iced#1415] [Q&A] [answered]
I'm making an app that requires global keyboard events (not only when the window is focused)

is there a way to do that in iced?

or maybe use an external crate and integrate with it?

**Comment by 13r0ck** [comment, marked as answer]
Use another crate that asks the OS for keyboard inputs, then create a subscription to convert those inputs to iced messages
See the websockey example for a how to, your subscription will probably be much simpler

**Reply by BKSalman** [reply]
yep! I managed to do that, but thanks for replying!
